### DISCUSSION TITLE
If we call supabase postgres from a aws resource in the same region, is it considered data egress?

### DISCUSSION THREAD

**Discussion by ShravanSunder** [supabase/supabase#16832] [Questions]
If we call supabase postgres from a aws resource in the same region, is it considered data egress? 
is there a definition of egress for supabase? is it the same as aws's definiiton?

**Comment by ghost** [comment]
I'm wondering the same thing. Is it worth trying to host my backend servers on AWS or should I just use Google CloudRun, DigitalOcean App Platform or whatever else I want?

**Comment by ShravanSunder** [comment]
@kiwicopple would be great to have some clarification or something in the docs about this

**this is what supabase Ai says**
> If you call Supabase Postgres from an AWS resource in the same region, it is not considered data egress. Supabase defines egress as a total quota that can be used for anything, including storage, realtime, auth, functions, or database. The organization receives a total egress quota, which varies depending on the plan. For example, the Pro plan has a quota of 250GB. You can view the pricing and free quota allowance of each line item on the pricing page.

> Supabase charges for certain usage line items in your project, including database size, storage size, egress, storage image transformations, realtime concurrent peak connections, realtime messages, edge function invocations, and the number of edge functions. You can monitor your usage and costs using the Usage section of your organization in the dashboard. This will show you how much you have spent so far in the current billing cycle, as well as an estimate of your expected total cost for the cycle based on your current usage.

> Supabase's definition of egress may not be the same as AWS's definition. However, Supabase provides an option to specify the region when invoking the Function. If your Function performs lots of database or storage operations, invoking the Function in the same region as your database may provide better performance.
